appcues/src/main/java/com/appcues/statemachine/Transitions.kt

```diff
@@ -101,7 +101,7 @@ internal interface Transitions {
             // this means the AppcuesActivity was destroyed externally (i.e. deep link) and we should
             // immediately transition to EndingExperience - not rely on the UI to do it for us (it's gone)
             Transition(
-                state = EndingStep(experience, flatStepIndex, null),
+                state = EndingStep(experience, flatStepIndex, action.markComplete, null),
```

Comment:
in this case, and line 115, the step is ending because of the EndExperience action - and this is where we need to actually detect whether its been marked completed, or is on the last step -- the skippable trait will not mark it completed, giving the desired behavior of not triggering step_completed if skipped.